### PR TITLE
Feature/handle invalid filenames

### PR DESCRIPTION
**Pull request by willread**
Handle invalid filesnames

Use the `ValidFilesMixin` from `rise-common-component` to validate the filenames provided via the `files` attribute.

### CODE REVIEW

File: src/rise-video.js [hunk 1]
```diff
@@ -1,33 +1,57 @@
...
     return html`
+      <style>
+        :host {
+          display: inline-block;
+          overflow: hidden;
+          position: relative;
+        }
+      </style>
       <h1>VIDEO</h1>
+        <template is="dom-repeat" items="[[_validFiles]]">
```
Comment (stulees) [inline src/rise-video.js]: @willread I'm presuming this is temporary markup here, correct?

File: src/rise-video.js [hunk 2]
```diff
@@ -1,33 +1,57 @@
...
       files: {
-        type: String,
-        value: ""
+        type: Array,
+        value: () => {
+          return [];
+        }
+      },
+
+      _validFiles: {
```
Comment (stulees) [inline src/rise-video.js]: @willread Why is `_validFiles` a surfaced  property of the component and not an internal variable?

File: src/rise-video.js (hunk 1 above)
Comment (willread) [reply src/rise-video.js]: @stulees Yes, it's just there to have some visual feedback until we work on the main functionality of the component.

File: src/rise-video.js (hunk 2 above)
Comment (willread) [reply src/rise-video.js]: @stulees Just so I could use it with dom-repeat. Maybe i'll just remove that test code, I don't think it's necessary at this point. Will update PR shortly.

File: src/rise-video.js [hunk 3]
```diff
@@ -1,33 +1,57 @@
...
+  }
+
   constructor() {
     super();
 
     this._setVersion( version );
 
     this._initialStart = true;
+    this._filesList = [];
+    this._validFileTypes = [ "mp4" ];
```
Comment (stulees) [inline src/rise-video.js]: @willread Valid file types for video are `webm` and `mp4`, thanks

**Comment by willread** [comment]
@stulees Pushed those changes

**Pull request merged**